src/utils.ts

```diff
@@ -68,107 +56,8 @@ const promptLayerApiRequest = async (body: TrackRequest) => {
   return body.request_response;
 };
 
-const promptLayerAllPromptTemplates = async (params?: Pagination) => {
-  const url = new URL(`${URL_API_PROMPTLAYER}/rest/prompts`);
-  Object.entries(params || {}).forEach(([key, value]) =>
-    url.searchParams.append(key, value)
-  );
-  try {
-    const response = await fetch(url, {
-      method: "GET",
-      headers: {
-        "Content-Type": "application/json",
-        "X-API-KEY": getApiKey(),
-      },
-    });
-    const data = await response.json();
-    if (response.status !== 200) {
-      console.warn(
-        `WARNING: While fetching all prompt templates, PromptLayer had the following error: ${JSON.stringify(
-          data
-        )}`
-      );
-    }
-    return data;
-  } catch (e) {
-    console.warn(
-      `WARNING: While fetching all prompt templates, PromptLayer had the following error: ${e}`
-    );
-  }
-};
-
-/**
- * Get a prompt from the PromptLayer library
- * @param prompt_name name of the prompt to get
- * @param api_key your api key
- * @param version version of the prompt to get, None for latest
- * @param label The release label of a prompt you want to get. Setting this will supercede version
- */
-const promptLayerGetPrompt = async (body: GetPromptTemplate) => {
-  const params: Record<string, string> = {
-    prompt_name: body.prompt_name,
-    version: body.version?.toString() ?? "",
-    label: body.label ?? "",
-  };
-  const url = new URL(`${URL_API_PROMPTLAYER}/library-get-prompt-template`);
-  url.search = new URLSearchParams(params).toString();
-  let response: Response;
-  try {
-    response = await fetch(url.toString(), {
-      method: "GET",
-      headers: {
-        "X-API-KEY": getApiKey(),
-      },
-    });
-  } catch (e) {
-    throw new Error(
-      `PromptLayer had the following error while getting your prompt: ${e}`
-    );
-  }
-  const data = await response.json();
-  if (response.status !== 200) {
-    throwOnBadResponse(
-      data,
-      `PromptLayer had the following error while retrieving your prompt template`
-    );
-  }
-  return data;
-};
-
-const promptLayerPublishPrompt = async (
-  body: LegacyPublishPromptTemplate
-): Promise<boolean> => {
-  let response: Response;
-  try {
-    response = await fetch(
-      `${URL_API_PROMPTLAYER}/library-publish-prompt-template`,
-      {
-        method: "POST",
-        headers: {
-          "Content-Type": "application/json",
-        },
-        body: JSON.stringify({
-          ...body,
-          api_key: getApiKey(),
-        }),
-      }
-    );
-  } catch (e) {
-    throw new Error(
-      `PromptLayer had the following error while publishing your prompt template: ${e}`
-    );
-  }
-  const data = await response.json();
-  if (response.status !== 200) {
-    throwOnBadResponse(
```

Comment:
We can remove the `throwOnBadResponse` function. It's dead code now, not used anywhere else.

Reply:
@buckyroberts good catch. Will remove it as well.